React Native v0.61 android build issue

**Describe the bug**
Unable to build android app with RN 0.61

**To Reproduce**
Steps to reproduce the behavior:
1. npx react-native init teststripe --version 0.61.5
2. yalc add @stripe/stripe-react-native
3. yarn install
4. npx react-native run-android
5. See error below
```
> Task :stripe_stripe-react-native:compileDebugKotlin FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.5/userguide/command_line_interface.html#sec:command_line_warnings
19 actionable tasks: 19 executed
e: /Users/obernal/code/test-stripe-rn/teststripe/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/StripeSdkCardViewManager.kt: (8, 38): Unresolved reference: DevSupportManagerBase

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_stripe-react-native:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 26s

error Failed to install the app. Make sure you have the Android development environment set up: https://facebook.github.io/react-native/docs/getting-started.html#android-development-environment. Run CLI with --verbose flag for more details.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
e: /Users/obernal/code/test-stripe-rn/teststripe/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/StripeSdkCardViewManager.kt: (8, 38): Unresolved reference: DevSupportManagerBase

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_stripe-react-native:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 26s

    at checkExecSyncError (child_process.js:616:11)
    at execFileSync (child_process.js:634:13)
    at runOnAllDevices (/Users/obernal/code/test-stripe-rn/teststripe/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:94:39)
    at buildAndRun (/Users/obernal/code/test-stripe-rn/teststripe/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:179:41)
    at then.result (/Users/obernal/code/test-stripe-rn/teststripe/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:133:12)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Expected behavior**
Build should be successful and the android app should start.

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.4
 - Browser [e.g. chrome, safari]
 - React Native version 0.61.5
 - Node version 10.15.0
 - Stripe React Native Version [commit 6346cb7884f875ae6c903c0efac6b7c00c19d879]

**Additional context**

Looks like `DevSupportManager` was added only added here https://github.com/facebook/react-native/commit/3f13c9abe6444ce8af75a98fb06df9a52293d653#diff-c8f8be6a9b0e7a9289d8c02feee7954425a90e1be133a48f80144230710889af which is tagged as `0.64.0` 

Everything on iOS works great so far. Is the library only compatible with React Native `0.64` for Android? 

Thanks!

--- a/android/src/main/java/com/reactnativestripesdk/StripeSdkCardViewManager.kt
+++ b/android/src/main/java/com/reactnativestripesdk/StripeSdkCardViewManager.kt
@@ -5,7 +5,6 @@ import com.facebook.react.bridge.ReadableNativeMap
 import com.facebook.react.bridge.WritableMap
 import com.facebook.react.bridge.WritableNativeMap
 import com.facebook.react.common.MapBuilder
-import com.facebook.react.devsupport.DevSupportManagerBase
 import com.facebook.react.modules.core.ExceptionsManagerModule
 import com.facebook.react.uimanager.SimpleViewManager
 import com.facebook.react.uimanager.ThemedReactContext

